src/main/scala/temple/generate/database/PostgresContext.scala

```diff
@@ -0,0 +1,10 @@
+package temple.generate.database
+
+case class PostgresContext(preparedType: PreparedType) extends DatabaseContext
+
+sealed trait PreparedType
+
+object PreparedType {
+  case object QuestionMarks extends PreparedType
+  case object DollarNumbers extends PreparedType
+}
```

Comment:
This also doesn't feel like it should be in this file? Not sure

Reply:
I'd argue this is pretty specific to the Postgres Context? or at least to SQL contexts hmm

Reply:
Nothing other than the main class/trait/object and its companion object should be at the root of a file. If `PreparedType` is directly related to `PostgresContext`, put it inside its companion object, otherwise it belongs in its own file